components/EventForm/EventForm.tsx

```
@@ -44,127 +46,137 @@ const EventForm: FC<EventFormPropsType> = ({
+                <View className="flex-1 space-y-0.5" key={file.url}>
+                  <DocumentTile
+                    name={file.name}
+                    url={file.url}
+                    mimeType={
+                      initialValues?.documents?.find(
+                        (doc) => doc.path === file.fullPath,
+                      )?.type
+                    }
+                    uploadingPercentage={
+                      uploadProgress &&
+                      uploadProgress[formatFileName(file.name)]
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)